stripper/ze_mars_escape_gp1.cfg

```diff
@@ -0,0 +1,12 @@
+;Lower roundtime from the default 10 minutes so players dont end up spending 5 minutes in endroom due to both CTs and Ts refusing to rush the other
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+	}
+	insert:
+	{
+		"OnMapSpawn" "commandCommandmp_roundtime 70-1""
```

Comment:
I'd like to leave a fair bit of time for things to play out initially at least, 8 minutes would work better I think.